scripts/rollup/modules.js

```diff
@@ -97,7 +101,7 @@ function getNodeModules(bundleType, isRenderer) {
       return {
         // Bundle object-assign once in the isomorphic React, and then use
         // that from the renderer UMD. Avoids bundling it in both UMDs.
-        'object-assign': isRenderer
+        'object-assign': moduleType === RENDERER
```

Comment:
I think we should instead check for `ISOMORPHIC` here, and reverse the ternary. Isomorphic gets the real thing, everyone else gets the shim. Even though technically we don’t have reconciler UMDs anyway.